HW10/IhorBohush/10-1_Practical_task_3_Bohush.py

```diff
@@ -0,0 +1,44 @@
+# Create an employee class. Each employee has characteristics such as name
+# and salary. The class should have a counter that calculates the total number of
+# employees, as well as a method that prints the total number of employees and a
+# method that displays information about each employee in particular, namely the
+# name and salary.
+# In addition to creating a class, display information about the base classes from which
+# the employee class is inherited (__base__), the class namespace (__dict__), the
+# class name (__name__), the module name in which the class is defined
+# (__module__), the documentation bar ( __doc__)
+
+
+class Employee:
+    """This is a class named Employee, that creates objects with name and salary of
+    employees. Also, there is a counter that calculates total number of employees"""
+
+    __counter = 0
+
+    def __init__(self, name, salary):
```

Comment:
In the init method, we should only initialize instance variables, not define methods